pages/api/sessionClient.ts

```diff
@@ -0,0 +1,39 @@
+﻿import {assertTokenIsValid} from "../../helpers/tokenHelpers";
```

Comment:
Likewise, could this also be moved out of `pages`?

Reply:
In fact - looks like we have some duplication between these 2 api clients.
Can we commonise them?